### PR TITLE
Update Clothing Assignments of Suit/Skirt

### PR DESCRIPTION
**Pull request by Tirochora**
# Description

Changes some of the parent clothing for various outfits in the game, such as the colored jumpskirts, the monastic robes, the operative jumpskirt, and miscellaneous clothing pieces that have open bottoms such as the gladiator's outfit and the kimono.

---

# Changelog

:cl: 
- fix: basic colored jumpskirts, kimonos, monastic robes, and the operative jumpskirt all now have the jumpskirt parent (operative jumpskirt is still unsensored), meaning they should be able to be worn by harpies.

### CODE REVIEW

**Review by FoxxoTrystan — CHANGES_REQUESTED** [review]
This is going to be a nightmare with conflict merges.

Please add comments on changes
and move all new jumpskirts to the floof directory wil commenting the old ones instead of deleting them.

**Comment by Tirochora** [comment]
> This is going to be a nightmare with conflict merges.
> 
> Please add comments on changes and move all new jumpskirts to the floof directory wil commenting the old ones instead of deleting them.

I'm not quite sure what you mean with 'moving all new jumpskirts to the floof directory, as there are no 'new' jumpskirts. Should I make and move the ones I modified to a separate 'jumpskirts.yml' in the floof directory?

File: Resources/Prototypes/Entities/Clothing/Uniforms/jumpskirts.yml [hunk 1]
```diff
@@ -9,6 +9,66 @@
...
+- type: entity
+  parent: ClothingUniformSkirtBase
+  id: ClothingUniformJumpsuitKimono
+  name: kimono
+  description: traditional chinese clothing
+  components:
+  - type: Sprite
+    sprite: Clothing/Uniforms/Jumpskirt/kimono.rsi
+  - type: Clothing
+    sprite: Clothing/Uniforms/Jumpskirt/kimono.rsi
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Entities/Clothing/Uniforms/jumpskirts.yml:70]: This need to be moved to a floof directory instead please.

File: Resources/Prototypes/Entities/Clothing/Uniforms/color_jumpskirts.yml [hunk 2]
```diff
@@ -1,6 +1,7 @@
+# Every jumpskirt in this file has been changed from having ClothingUniformBase to having ClothingUniformSkirtBase as a parent.
 # White Jumpskirt
 - type: entity
-  parent: ClothingUniformBase
+  parent: ClothingUniformSkirtBase
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Entities/Clothing/Uniforms/color_jumpskirts.yml:4]:
```suggestion
  parent: ClothingUniformSkirtBase # FloofStation
```

File: Resources/Prototypes/Entities/Clothing/Uniforms/color_jumpskirts.yml [hunk 3]
```diff
@@ -27,7 +28,7 @@
 
 # Grey Jumpskirt
 - type: entity
-  parent: ClothingUniformBase
+  parent: ClothingUniformSkirtBase
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Entities/Clothing/Uniforms/color_jumpskirts.yml:31]:
```suggestion
  parent: ClothingUniformSkirtBase # FloofStation
```

File: Resources/Prototypes/Entities/Clothing/Uniforms/color_jumpskirts.yml [hunk 4]
```diff
@@ -58,7 +59,7 @@
 
 # Black Jumpskirt
 - type: entity
-  parent: ClothingUniformBase
+  parent: ClothingUniformSkirtBase
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Entities/Clothing/Uniforms/color_jumpskirts.yml:62]:
```suggestion
  parent: ClothingUniformSkirtBase # FloofStation
```

File: Resources/Prototypes/Entities/Clothing/Uniforms/color_jumpskirts.yml [hunk 5]
```diff
@@ -89,7 +90,7 @@
 
 # Blue Jumpskirt
 - type: entity
-  parent: ClothingUniformBase
+  parent: ClothingUniformSkirtBase
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Entities/Clothing/Uniforms/color_jumpskirts.yml:93]:
```suggestion
  parent: ClothingUniformSkirtBase # FloofStation
```

File: Resources/Prototypes/Entities/Clothing/Uniforms/color_jumpskirts.yml [hunk 6]
```diff
@@ -120,7 +121,7 @@
 
 # Dark Blue Jumpskirt
 - type: entity
-  parent: ClothingUniformBase
+  parent: ClothingUniformSkirtBase
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Entities/Clothing/Uniforms/color_jumpskirts.yml:124]:
```suggestion
  parent: ClothingUniformSkirtBase # FloofStation
```

File: Resources/Prototypes/Entities/Clothing/Uniforms/color_jumpskirts.yml [hunk 7]
```diff
@@ -151,7 +152,7 @@
 
 # Teal Jumpskirt
 - type: entity
-  parent: ClothingUniformBase
+  parent: ClothingUniformSkirtBase
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Entities/Clothing/Uniforms/color_jumpskirts.yml:155]:
```suggestion
  parent: ClothingUniformSkirtBase # FloofStation
```

File: Resources/Prototypes/Entities/Clothing/Uniforms/color_jumpskirts.yml [hunk 8]
```diff
@@ -182,7 +183,7 @@
 
 # Green Jumpskirt
 - type: entity
-  parent: ClothingUniformBase
+  parent: ClothingUniformSkirtBase
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Entities/Clothing/Uniforms/color_jumpskirts.yml:186]:
```suggestion
  parent: ClothingUniformSkirtBase # FloofStation
```

File: Resources/Prototypes/Entities/Clothing/Uniforms/color_jumpskirts.yml [hunk 9]
```diff
@@ -213,7 +214,7 @@
 
  # Dark Green Jumpskirt
 - type: entity
-  parent: ClothingUniformBase
+  parent: ClothingUniformSkirtBase
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Entities/Clothing/Uniforms/color_jumpskirts.yml:217]:
```suggestion
  parent: ClothingUniformSkirtBase # FloofStation
```

File: Resources/Prototypes/Entities/Clothing/Uniforms/color_jumpskirts.yml [hunk 10]
```diff
@@ -244,7 +245,7 @@
 
 # Orange Jumpskirt
 - type: entity
-  parent: ClothingUniformBase
+  parent: ClothingUniformSkirtBase
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Entities/Clothing/Uniforms/color_jumpskirts.yml:248]:
```suggestion
  parent: ClothingUniformSkirtBase # FloofStation
```

File: Resources/Prototypes/Entities/Clothing/Uniforms/color_jumpskirts.yml [hunk 11]
```diff
@@ -275,7 +276,7 @@
 
 # Pink Jumpskirt
 - type: entity
-  parent: ClothingUniformBase
+  parent: ClothingUniformSkirtBase
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Entities/Clothing/Uniforms/color_jumpskirts.yml:279]:
```suggestion
  parent: ClothingUniformSkirtBase # FloofStation
```

File: Resources/Prototypes/Entities/Clothing/Uniforms/color_jumpskirts.yml [hunk 12]
```diff
@@ -306,7 +307,7 @@
 
 # Red Jumpskirt
 - type: entity
-  parent: ClothingUniformBase
+  parent: ClothingUniformSkirtBase
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Entities/Clothing/Uniforms/color_jumpskirts.yml:310]:
```suggestion
  parent: ClothingUniformSkirtBase # FloofStation
```

File: Resources/Prototypes/Entities/Clothing/Uniforms/color_jumpskirts.yml [hunk 13]
```diff
@@ -337,7 +338,7 @@
 
 # Yellow Jumpskirt
 - type: entity
-  parent: ClothingUniformBase
+  parent: ClothingUniformSkirtBase
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Entities/Clothing/Uniforms/color_jumpskirts.yml:341]:
```suggestion
  parent: ClothingUniformSkirtBase # FloofStation
```

File: Resources/Prototypes/Entities/Clothing/Uniforms/color_jumpskirts.yml [hunk 14]
```diff
@@ -368,7 +369,7 @@
 
 # Purple Jumpskirt
 - type: entity
-  parent: ClothingUniformBase
+  parent: ClothingUniformSkirtBase
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Entities/Clothing/Uniforms/color_jumpskirts.yml:372]:
```suggestion
  parent: ClothingUniformSkirtBase # FloofStation
```

File: Resources/Prototypes/Entities/Clothing/Uniforms/color_jumpskirts.yml [hunk 15]
```diff
@@ -399,7 +400,7 @@
 
 # Light Brown Jumpskirt
 - type: entity
-  parent: ClothingUniformBase
+  parent: ClothingUniformSkirtBase
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Entities/Clothing/Uniforms/color_jumpskirts.yml:403]:
```suggestion
  parent: ClothingUniformSkirtBase # FloofStation
```

File: Resources/Prototypes/Entities/Clothing/Uniforms/color_jumpskirts.yml [hunk 16]
```diff
@@ -430,7 +431,7 @@
 
 # Brown Jumpskirt
 - type: entity
-  parent: ClothingUniformBase
+  parent: ClothingUniformSkirtBase
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Entities/Clothing/Uniforms/color_jumpskirts.yml:434]:
```suggestion
  parent: ClothingUniformSkirtBase # FloofStation
```

File: Resources/Prototypes/Entities/Clothing/Uniforms/color_jumpskirts.yml [hunk 17]
```diff
@@ -461,7 +462,7 @@
 
 # Maroon Jumpskirt
 - type: entity
-  parent: ClothingUniformBase
+  parent: ClothingUniformSkirtBase
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Entities/Clothing/Uniforms/color_jumpskirts.yml:465]:
```suggestion
  parent: ClothingUniformSkirtBase # FloofStation
```

File: Resources/Prototypes/Entities/Clothing/Uniforms/jumpsuits.yml [hunk 18]
```diff
@@ -46,16 +46,17 @@
...
-  parent: ClothingUniformBase
-  id: ClothingUniformJumpsuitBartenderPurple
-  name: purple bartender's uniform
-  description: A special purple outfit to serve drinks.
-  components:
-  - type: Sprite
-    sprite: Clothing/Uniforms/Jumpsuit/bartender_purple.rsi
-  - type: Clothing
-    sprite: Clothing/Uniforms/Jumpsuit/bartender_purple.rsi
+# Moved this to jumpskirts.yml, further modifications made there.
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Entities/Clothing/Uniforms/jumpsuits.yml:49]:
```suggestion
# Moved this to jumpskirts.yml, further modifications made there. // FloofStation
```

File: Resources/Prototypes/Entities/Clothing/Uniforms/jumpsuits.yml [hunk 19]
```diff
@@ -396,16 +397,17 @@
...
-  parent: ClothingUniformBase
-  id: ClothingUniformJumpsuitKimono
-  name: kimono
-  description: traditional chinese clothing
-  components:
-  - type: Sprite
-    sprite: Clothing/Uniforms/Jumpsuit/kimono.rsi
-  - type: Clothing
-    sprite: Clothing/Uniforms/Jumpsuit/kimono.rsi
+# Moved this to jumpskirts.yml, further modifications made there.
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Entities/Clothing/Uniforms/jumpsuits.yml:400]:
```suggestion
# Moved this to jumpskirts.yml, further modifications made there. // FloofStation
```

File: Resources/Prototypes/Entities/Clothing/Uniforms/jumpsuits.yml [hunk 20]
```diff
@@ -892,27 +894,29 @@
...
-  parent: ClothingUniformBase
-  id: ClothingUniformJumpsuitMonasticRobeLight
-  name: light monastic robe
-  description: It's a light robe, often worn by religious folk.
-  components:
-    - type: Sprite
-      sprite: Clothing/Uniforms/Jumpsuit/monastic_robe_light.rsi
-    - type: Clothing
-      sprite: Clothing/Uniforms/Jumpsuit/monastic_robe_light.rsi
+# Moved this to jumpskirts.yml, further modifications made there.
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Entities/Clothing/Uniforms/jumpsuits.yml:897]:
```suggestion
# Moved this to jumpskirts.yml, further modifications made there. // FloofStation
```

File: Resources/Prototypes/Entities/Clothing/Uniforms/jumpsuits.yml [hunk 21]
```diff
@@ -1191,16 +1195,17 @@
...
-  parent: ClothingUniformBase
-  id: ClothingUniformJumpsuitGladiator
-  name: gladiator uniform
-  description: Made for true gladiators (or Ash Walkers).
-  components:
-  - type: Sprite
-    sprite: Clothing/Uniforms/Jumpsuit/gladiator.rsi
-  - type: Clothing
-    sprite: Clothing/Uniforms/Jumpsuit/gladiator.rsi
+# Moved this to jumpskirts.yml, further modifications made there.
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Entities/Clothing/Uniforms/jumpsuits.yml:1198]:
```suggestion
# Moved this to jumpskirts.yml, further modifications made there. // FloofStation
```

File: Resources/Prototypes/Entities/Clothing/Uniforms/specific.yml [hunk 22]
```diff
@@ -1,5 +1,5 @@
 ﻿- type: entity
-  parent: ClothingUniformBase
+  parent: ClothingUniformSkirtBase
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Entities/Clothing/Uniforms/specific.yml:2]:
```suggestion
  parent: ClothingUniformSkirtBase # FloofStation
```

File: Resources/Prototypes/Entities/Clothing/Uniforms/base_clothinguniforms.yml [hunk 23]
```diff
@@ -36,6 +36,11 @@
   - type: Clothing
     slots: [innerclothing]
     femaleMask: UniformTop
+# Adds the 'skirt' tag to UnsensoredClothingUniformSkirtBase to, well, make it a skirt.
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Entities/Clothing/Uniforms/base_clothinguniforms.yml:39]:
```suggestion
# Adds the 'skirt' tag to UnsensoredClothingUniformSkirtBase to, well, make it a skirt. // FloofStation
```

File: Resources/Prototypes/Entities/Clothing/Uniforms/base_clothinguniforms.yml [hunk 24]
```diff
@@ -66,6 +71,6 @@
     slots: [innerclothing]
     femaleMask: UniformTop
   - type: Tag #DeltaV, needed for species with nonhuman legs/can only wear skirts
+# Removed the subsequent 'skirt' tag from ClothingUniformSkirtBase, as the unsensored parent now has it.
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Entities/Clothing/Uniforms/base_clothinguniforms.yml:74]:
```suggestion
# Removed the subsequent 'skirt' tag from ClothingUniformSkirtBase, as the unsensored parent now has it. // FloofStation
```
Also doing this kinda make this base now worthless.

**Review by FoxxoTrystan — CHANGES_REQUESTED** [review]

**Comment by Tirochora** [comment]
I forgot to close this earlier - I brought this up to EE, as there's no reason not to and it makes things simpler.

**Pull request closed without merging**